### PR TITLE
Added checks if config is set correctly

### PR DESCRIPTION
**Pull request by Nitwel**
No more erros when an option is not set.

### CODE REVIEW

File: src/layouts/timeline/layout.vue [hunk 1]
```diff
@@ -49,22 +48,31 @@ export default {
...
+
+        if (!this.viewOptions.date) return;
 
         var date = new Date(item[this.viewOptions.date].substr(0, 10) + "T00:00:00");
         var existingDay = this.$lodash.find(days, { date: date });
 
-        var color = item[this.viewOptions.color];
-        if (this.fields[this.viewOptions.color].field == "action") {
-          color = this.actionColor[color];
+        var color = null;
```
Comment (rijkvanzanten) [inline src/layouts/timeline/layout.vue]:
```suggestion
        let color = null;
```

File: src/layouts/timeline/layout.vue [hunk 2]
```diff
@@ -49,22 +48,31 @@ export default {
...
+
+          if (
+            this.fields[this.viewOptions.color] &&
+            this.fields[this.viewOptions.color].field == "action"
+          ) {
+            color = this.actionColor[color];
+          }
         }
 
+        var contentType = this.viewOptions.content ? this.fields[this.viewOptions.content] : null;
```
Comment (rijkvanzanten) [inline src/layouts/timeline/layout.vue]:
```suggestion
        const contentType = this.viewOptions.content ? this.fields[this.viewOptions.content] : null;
```

File: src/layouts/timeline/layout.vue [hunk 3]
```diff
@@ -49,22 +48,31 @@ export default {
...
+
         var event = {
```
Comment (rijkvanzanten) [inline src/layouts/timeline/layout.vue]:
```suggestion
        const event = {
```

File: src/layouts/timeline/layout.vue (hunk 2 above)
Comment (Nitwel) [reply src/layouts/timeline/layout.vue]: Will this change when changing the field in the options?

**Pull request merged**